app/views/admin/categories/index.html.erb

```diff
@@ -6,9 +6,10 @@
         <ul class="categories">
             <% @categories.each do |category| %>
                 <li>
-                    <%= link_to category.name, category %>
```

Comment:
Put this back, and change .name to .title since that is what you are trying to display.

Reply:
also for the link use the `admin/categories#show` prefix and put the `category` in a parenthesis -> `(category)`
![Screen Shot 2021-05-06 at 8 30 24 PM](https://user-images.githubusercontent.com/74635797/117298480-fd75e480-aea9-11eb-8247-0ce9658e6dd4.png)